gradio/chat_interface.py

```diff
@@ -461,18 +461,36 @@ def _generate_chat_title(self, conversation: list[MessageDict]) -> str:
             title = title[:40] + "..."
         return title or "Conversation"
 
+    def make_serializable(self, obj: Any) -> dict | list | str:
```

Comment:
This can be a static method

Reply:
Since its recursive I would have to call itself using `ChatInterface._make_serializable` instead of using self. opted not to since we don't do that anywhere else.

Reply:
No, it's not defined

Reply:
Ok but if you do the inner recursive function approach suggested below, that should definitely work, right?

Reply:
yeah fixed it